File: simple-example-of-using-android-dagger2-java-version/app/src/main/java/com/example/simpleexampleofusingandroiddagger2javaversion/MainActivity.java
package com.example.simpleexampleofusingandroiddagger2javaversion;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;
import javax.inject.Inject;
import dagger.android.AndroidInjection;

public class MainActivity extends AppCompatActivity {

    @Inject
    MainActivityViewModel mainActivityViewModel;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        AndroidInjection.inject(this);

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView textView = findViewById(R.id.tv);
        textView.setText(mainActivityViewModel.someData);

    }
}

File: mvvm-todo-list-app-room-dependency/app/src/main/java/com/example/mvvmtodolistappprojectsetup/ui/MainActivity.kt
package com.example.mvvmtodolistappprojectsetup.ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.mvvmtodolistappprojectsetup.R
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}